_posts/2018-07-25-Relay-Networking-Deep-Dive.markdown

```diff
@@ -131,15 +155,31 @@ const cacheHandler = async (
 };
 ```
 
-Built on top of the other 2 implementations, we use RelayQueryResponseCache to query GraphQL requests based on query
-and variables. Every time a mutation is done, we should invalidate all cache as we are not sure how this affect all
-cached queries responses.
-
 ### Network using Observable
 
-Relay provides a limited implementation of [ESObservables][]. I recommend reading [A General Theory of
-Reactivity][reactivity] to understand why we need to use Observable instead of promises in some situations. While a
-promise is one value in a time space, an observable is a stream of values in a time space.
+Relay provides a limited implementation of the upcoming [ESObservables][] spec. I recommend reading [A General
+Theory of Reactivity][reactivity] to understand why Observables are a great solution instead of promises in some
+situations. Notably; a promise is one value in a time space, an observable is a stream of values in a time space.
+
+[TODO: Why Sink and not the Relay Observable? Observable is exported but has one more function (complete)]
```

Comment:
The `Sink` here threw me off.

From reading the APIs, could this be a relay Observable instead of a Sink? They call it that in the API. Then there's no new terminology to learn for someone coming through, nor the need to add the `Sink` interface to your codebase